src/components/KeyAgencyActions.vue

```diff
@@ -79,10 +79,23 @@ export default Vue.extend({
       if (this.answer[0].id === undefined) return;
 
       this.displayedJustification = this.answer[0].justify_answer;
-      if (this.answer[0].short_answer === "Y") {
-        this.yesNoValue = this.items[0];
-      } else {
-        this.yesNoValue = this.items[1];
+      if (this.answer[0].selected_choice === "") {
```

Comment:
Please replace the 'yesNoValue' variable to 'agreementValue' (or anything more descriptive).